dpnp/dpnp_iface_histograms.py

```diff
@@ -751,6 +755,149 @@ def histogram_bin_edges(a, bins=10, range=None, weights=None):
     return bin_edges
 
 
+def histogram2d(x, y, bins=10, range=None, density=None, weights=None):
+    # pylint: disable=line-too-long
+    """
+    Compute the bi-dimensional histogram of two data samples.
+
+    Parameters
+    ----------
+    x : {dpnp.ndarray, usm_ndarray} of shape (N,)
+        An array containing the x coordinates of the points to be
+        histogrammed.
+    y : {dpnp.ndarray, usm_ndarray} of shape (N,)
+        An array containing the y coordinates of the points to be
+        histogrammed.
+    bins : {int, list of dpnp.ndarray or usm_ndarray, sequence of scalars}, optional
```

Comment:
based on the below description it is more like:
```suggestion
    bins : {int, dpnp.ndarray, usm_ndarray, [int, int], [array, array], [int, array], [array, int]}, optional
```